tsconfig.json

```diff
@@ -20,12 +20,13 @@
             "node",
             "mocha"
         ],
-        "resolveJsonModule": true
+        "resolveJsonModule": true,
+        "esModuleInterop": true,
```

Comment:
I am not sure why do we need this? Do we have a module that we want to default import however it has no default export?

Reply:
I read somewhere that we need to use this when `"module": "commonjs"`

Reply:
Really? What happens if we remove it?

Reply:
Removed this, and it worked, thanks 👍